Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -621,7 +621,7 @@ BoardInit (
       VariableConstructor (VarBase, VarSize);
     }
 
-    if (PcdGetBool (PcdSmmRebaseEnabled)) {
+    if (PcdGet8 (PcdSmmRebaseMode) != SMM_REBASE_DISABLE) {
```

Comment:
Can we remove this condition?   I think we should already provide correct TSEG base/size regardless of SBL SMM rebase enable/disable.   Even SBL disables SMM,  payload might still want to enable it.  So TSEG base/size will still be useful.